app/uk/gov/hmrc/hmrcemailrenderer/templates/itr/ItrTemplates.scala

```diff
@@ -30,7 +30,14 @@ object ItrTemplates {
       subject = "Apply to use Venture Capital Schemes: verify your email address",
       plainTemplate = txt.itrEmailVerification.f,
       htmlTemplate = html.itrEmailVerification.f,
-      priority = Some(MessagePriority.Urgent))
+      priority = Some(MessagePriority.Urgent)),
+    MessageTemplate.create(
+      templateId = "investment_tax_relief_confirmation_email",
+      fromAddress = govUkTeamAddress,
+      service = InvestmentTaxRelief,
+      subject = "Apply to use Venture Capital Schemes: Advance Assurance application received",
+      plainTemplate = txt.itrEmailConfirmation.f,
+      htmlTemplate = html.itrEmailConfirmation.f)
```

Comment:
That depends on whether an email confirmation can be viewed as urgent? For obvious reasons verify is urgent, however I didn't think this would be the case with confirmation emails. Although, I did not think it could take up to a few hours so I will update the priority.